web/modules/custom/bglobal_angular/components/voices_list/template.html

```diff
@@ -1,6 +1,13 @@
-<div *ngFor="let post of posts" data-ng-cloak class="voices-section grid-flip" style="width: 400px; padding: 10px;">
-  <!-- {{ i }} {{ getKeys(post) }} -->
-  <img [src] = "post.image.src" [alt] = "post.image.alt" style="width: 400px;" class="voice-section--img m-b-20 img-fluid" />
-  <p class="marginless-p"><strong><a [href] = "post.path" class="blog-header">{{ post.title }}</a></strong></p>
```

Comment:
- Will this be the title of the post? If it really is a title, this should not be wrapped in `<p>`. 
It can either be a heading (h3, depending on what other headers are being used on the page), or even a <span>.
- Also, rather than wrapping the `<a>` in `<strong>`, since the link has a class, I would style the link in CSS to be bold.
- The class name should probably be something a little bit more granular - maybe `blog-header__link`?